src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,117 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private int size = 0;
+    private T[] arr;
+
+    public ArrayList() {
+        arr = (T[]) new Object[DEFAULT_CAPACITY];
+    }
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line at the start of the constructor.